src/services/accounts/AccountsValidateService.ts

```diff
@@ -26,16 +32,28 @@ export class AccountsValidateService extends AbstractService {
 
 		if (defaults.allowedEncodedLengths.includes(u8Address.length)) {
 			const [isValid, , , ss58Prefix] = checkAddressChecksum(u8Address);
-
+			const networkName = allNetworks.filter(
+				(network) => network.prefix === ss58Prefix
+			)[0].network;
+			if (isHex(address)) {
+				accountId = address;
+			} else {
+				const registry = new TypeRegistry();
+				accountId = registry.createType('AccountId', address).toHex();
```

Comment:
@TarikGul Not sure if this is the way to retrieve the `accountId` but still I pushed the change to get some feedback. 
The logic is described in the last commit message and I added below also : 
- If the address given is a hex value then it returns the same as the `accountId`
- If the address given is not a hex value then it returns the hex value of the related registry type

Reply:
See above comment on how we can properly generate the accountId with the correct registry. 

To address the above logic though. Seems fine to me 👍